skops/hub_utils/_hf_hub.py

```diff
@@ -327,6 +328,65 @@ def init(
         raise
 
 
+def add_files(files: Sequence[str | Path], dst: str | Path) -> None:
```

Comment:
we could also have the signature as:

```suggestion
def add_files(*files, dst):
```

makes it easier for people to just add many while adding one would be easier as:

```python
add_files(__file__, dst)
```